README.md

```diff
@@ -64,74 +64,80 @@ Please consider which installation method you prefer. Cloud Installation require
 
 Instructions are for an Ubuntu/Debian system. Varies slightly for mac/fedora/etc.
 
-- [MySQL](#MySQL)
-- [Ruby version manager: RVM / Rbenv](#Ruby-version-manager:-RVM-/-Rbenv)
-- [Package manager: Npm and Yarn](#Package-manager:-Npm-and-Yarn)
+1. [MySQL](#MySQL) - Database
+2. [RVM](#RVM) - Ruby version manager
+3. [Yarn](#Yarn) - Package manager
 
 #### MySQL
 
-- MacOS and Linux users, please reference [MYSQL.md](MYSQL.md) instead.
+MacOS and Linux users, please reference [MYSQL.md](MYSQL.md) instead.
 
-Install a database, if necessary. sqlite does not seem to work due to some table constraints:
+1. Install MYSQL:
 
 ```Bash
 $ sudo apt-get install mysql-server
 ```
 
+2. Application-specific dependencies:
 
-Application-specific dependencies:
-
-`$ sudo apt-get install bundler libmysqlclient-dev imagemagick ruby-rmagick libfreeimage3 libfreeimage-dev ruby-dev libmagickcore-dev libmagickwand-dev`
+```Bash
+$ sudo apt-get install bundler libmysqlclient-dev imagemagick ruby-rmagick libfreeimage3 libfreeimage-dev ruby-dev libmagickcore-dev libmagickwand-dev
+```
 
-(optional) For exporting, you'll need GDAL >=1.7.x (gdal.org), as well as `curl` and `zip`-- but these are not needed for much of development, unless you're working on the exporting features. 
+3. *(Optional)*: For exporting, you'll need GDAL >=1.7.x (gdal.org), as well as `curl` and `zip`-- but these are not needed for much of development, unless you're working on the exporting features. 
 
   ```Bash
   $ sudo apt-get install gdal-bin python-gdal curl libcurl4-openssl-dev libssl-dev zip
   ```
 
 ==================
 
+#### RVM
 
-#### Ruby version manager: RVM / Rbenv
+This is for RVM, but the alternative, **rbenv**, also works (instructions not listed here). Don't install RVM if you already have rbenv!
 
-This is for RVM, but the alternative, Rbenv, also works (instructions not listed here). Don't install RVM if you already have Rbenv!
-
-Install RVM for Ruby management (http://rvm.io)
+1. Install RVM: (http://rvm.io)
 
 ```Bash
 $ curl -L https://get.rvm.io | bash -s stable
 ```
 
-**Note:** At this point during the process, you may want to log out and log back in, or open a new terminal window; RVM will then properly load in your environment. 
+2. At this point during the process, you may want to log out and log back in, or open a new terminal window; RVM will then properly load in your environment. 
+
+   - *Ubuntu users only:* you may need to enable `Run command as a login shell` in Ubuntu's Terminal, under Profile Preferences > Title and Command. Then close the terminal and reopen it.
 
-**Ubuntu users:** You may need to enable `Run command as a login shell` in Ubuntu's Terminal, under Profile Preferences > Title and Command. Then close the terminal and reopen it.
+3. Use RVM to install version 2.4.6 of Ruby:
 
-Then, use RVM to install version 2.4.6 of Ruby:
   ```Bash
 $ rvm install 2.4.6
```

Comment:
We can remove this and add it in the installer so rvm doesn't install its default ruby 2.6
`curl -sSL https://get.rvm.io | bash -s stable --ruby=2.4.6`

Reply:
@kaustubh-nair What does this part of the command mean: `--ruby=ruby-2.4.6`? Does that set the new default to `2.4.6`? If so, would it be better to keep the current one (so new projects will use `2.6`) and use RVM how its meant, to manage multiple versions?

Reply:
Yes, instead of rvm installing 2.6 which we don't use at all, we let rvm use 2.4.6 as the default version.
It still doesn't change any rvm functionality. It can still be used to handle multiple versions.
It just decreases an extra step in the installation process. 
It's unnecessary for a user to have 2 ruby versions. They take up quite some space
But it's not really important so feel free to keep it as is!

Reply:
@kaustubh-nair I don't work with these things as much as you guys - @alaxalves can you give a 2nd opinion? Just want to confirm we do whats best for the contributor!!